src/IO/H5/Python/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ spectre_python_add_module(
   File.cpp
   TensorData.cpp
   VolumeData.cpp
+  PYTHON_FILES
+  ExtractInputSourceYamlFromH5.py
```

Comment:
Should this first be put in the `PYTHON_EXECUTABLES` and in the next commit move it to `PYTHON_FILES`?

Reply:
Doesn't really matter! Only affects whether or not the script is symlinked to bin/. I'll remove `PYTHON_EXECUTABLES` altogether once all scripts are added to the CLI.